python/cuml/test/dask/test_kmeans.py

```diff
@@ -161,8 +159,10 @@ def test_transform(nrows, ncols, nclusters, n_parts, input_type, client):
                                        stress_param(50)])
 @pytest.mark.parametrize("n_parts", [unit_param(None), quality_param(7),
                                      stress_param(50)])
+@pytest.mark.parametrize("SCORE_EPS", [unit_param(0.06), stress_param(35.0)])
 @pytest.mark.parametrize("input_type", ["dataframe", "array"])
-def test_score(nrows, ncols, nclusters, n_parts, input_type, client):
+def test_score(nrows, ncols, nclusters, n_parts,
+               input_type, SCORE_EPS, client):
```

Comment:
Very small nitpick- if we are going to make this a parametrizable argument, we probably want to change it from all caps. When I see all caps in the code like this I immediately assume it’s a constant defined somewhere.